Client/qtTeamTalk/mainwindow.cpp

```diff
@@ -682,6 +682,35 @@ void MainWindow::loadSettings()
 
     if(connect_ok)
         QTimer::singleShot(0, this, &MainWindow::slotConnectToLatest);
+
+#if defined(Q_OS_WINDOWS)
+    bool tolkLoaded = Tolk_IsLoaded();
+    if (!tolkLoaded)
+        Tolk_Load();
+
+    bool SRActive = Tolk_DetectScreenReader() != nullptr;
+
+    if (!tolkLoaded)
+        Tolk_Unload();
+
+    if (ttSettings->value(SETTINGS_GENERAL_FIRSTSTART, SETTINGS_GENERAL_FIRSTSTART_DEFAULT).toBool() && SRActive)

```

Comment:
I've made a modification so the message popup only comes with a screen-reader is detected